### ISSUE TITLE
Cornelsen Authentication not working

### ISSUE THREAD

**Issue by Jogaxell** [RythenGlyth/EbookDownloader#1]
The Cornelsen Downloader cannot authenticate itself. As it seems to me, the program fails to get the cornelsen-jwt cookie.

**Comment by RythenGlyth** [comment]
I'll try to look into it in the next few days.
If there is an error message in the console, it would be great if you could share it

**Comment by Jogaxell** [comment]
Sure. Here's the console log:

```
Could not get library - 754
TypeError: Cannot read properties of undefined (reading 'value')
    at F:\js\EbookDownloader - Copy\EbookDownloader.js:717:124
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Let me know if there's anything else you need.

**Comment by HPeters0512** [comment]
Should be fixed with commit 6873f6a

**Comment by RythenGlyth** [comment]
should be fixed with commit 6873f6a

**Comment by RythenGlyth** [comment]
The javascript adresses should now be loaded dynamically. Should be fixed as of committ 0e68467
